python/servo/package_commands.py

```diff
@@ -96,8 +96,11 @@ def copy_dependencies(binary_path, lib_path):
 
 def copy_windows_dependencies(binary_path, destination):
     deps = [
+        "api-ms-win-crt-runtime-l1-1-0.dll",
         "libcryptoMD.dll",
         "libsslMD.dll",
+        "msvcp140.dll",
+        "vcruntime140.dll",
```

Comment:
https://blogs.msdn.microsoft.com/vcblog/2015/03/03/introducing-the-universal-crt/